### ISSUE TITLE
Count keyword completion ends up at wrong place

### ISSUE THREAD

**Issue by ufuk-ergin-carbon** [dbeaver/dbeaver#22854]
### Description

Hi folks! I'm not sure if this is a feature request or this is a bug or even it's intentional on the part of the developers, but when I start typing "coun", obviously the code completion offers me COUNT. When I hit tab to accept the suggestion, it inserts the keyword along with the parentheses. But, the cursor stays at "T" of count, instead of being in the parenthesis. This is annoying since then I have to press right arrow key to be in the parenthesis. Would it not be better for the code completion to move into the parenthesis? Attaching a short screen recording for what I mean.

Thank you!

https://github.com/dbeaver/dbeaver/assets/114302087/5d022b85-9b15-4f34-8bd6-a9c4c31b431c

### DBeaver Version

Version 23.3.5.202402181738

### Operating System

MacOS Sonoma 14.3.1

### Database and driver

Server: Snowflake 8.6.2
Driver: Snowflake 3.14.4

**Comment by LonwoLonwo** [comment]
Hello @ufuk-ergin-carbon 

Unfortunately, I can't reproduce it.

How about other functions in the Snowflake? Is it reproducible with the 
`SELECT GETBIT()` statement? as an example?

**Comment by ufuk-ergin-carbon** [comment]
Hi @LonwoLonwo , it is also reproducible with `SELECT GETBIT()` statement. I am attaching a screen recording for your review. When you hit tab to complete the `COUNT`, for example, does the cursor jump to inside of the parenthesis? 

https://github.com/dbeaver/dbeaver/assets/114302087/f772ed00-f25b-4dcf-b3ab-cf1076943767

**Comment by LonwoLonwo** [comment]
Yes, my cursor jumps inside of the parenthesis for COUNT and other functions.

Could you please try to reset your UI settings? https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings

**Comment by E1izabeth** [comment]
It's been a while since no update here. If the issue is still actual please let me know and provide additional information.

**Comment by ufuk-ergin-carbon** [comment]
Hello, I just reset the user UI settings, and the issue is still there. Let me know if you need another screen recording.

**Comment by ufuk-ergin-carbon** [comment]
Hey folks! Can you reproduce this issue? Do you need another screen recording or suggestions for other steps to alleviate this problem?

**Comment by E1izabeth** [comment]
We passed this issue to QA team and wait for the answer, if they're able to reproduce it.

**Comment by ufuk-ergin-carbon** [comment]
Hello @E1izabeth , are there any updates regarding this? I can provide additional information if necessary. With today's update, the issue still persists.

**Comment by E1izabeth** [comment]
We have no updates regarding this issue

**Comment by ufuk-ergin-carbon** [comment]
FYI. Tested this problem on Windows, and I was able to replicate the issue there as well. Let me know if you need a screen recording on Windows.
